WinUIGallery/ControlPages/XamlFundamentals/TemplatesPage.xaml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="WinUIGallery.ControlPages.TemplatesPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:WinUIGallery"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <ItemsPanelTemplate x:Key="WrapGridTemplate">
+            <WrapGrid Orientation="Horizontal" />
+        </ItemsPanelTemplate>
+
+        <ItemsPanelTemplate x:Key="StackPanelTemplate">
+            <StackPanel Orientation="Vertical" />
+        </ItemsPanelTemplate>
+    </Page.Resources>
+
+    <StackPanel>
+        <StackPanel Spacing="12"
+                    Margin="0,12,0,0">
+            <RichTextBlock>
+                <Paragraph FontWeight="SemiBold">Where can you create templates?</Paragraph>
+                <Paragraph>You can define templates at the app, page, or control level, just like styles and resources. The placement depends on where and how you want the template to be reused.</Paragraph>
+            </RichTextBlock>
+
+            <RichTextBlock>
+                <Paragraph FontWeight="SemiBold">Key points:</Paragraph>
+                <Paragraph>- Templates allow you to redefine how a control is built visually while keeping its behavior intact.</Paragraph>
+                <Paragraph>- Templates can be defined at the App, Page, or Control level, depending on the desired scope.</Paragraph>
```

Comment:
Maybe combine this with the text above so it's just a single paragraph of text.

Reply:
I separated the text blocks to improve spacing.

Reply:
@niels9001
Can this be considered resolved?